metrics/resource_view_test.go

```diff
@@ -170,12 +171,14 @@ func TestAllMetersExpiration(t *testing.T) {
 
 	// Expire the second entry
 	fakeClock.Step(9 * time.Minute) // t+12m
-	time.Sleep(time.Second)         // Wait a second on the wallclock, so that the cleanup thread has time to finish a loop
-	allMeters.lock.Lock()
+	_ = wait.PollImmediate(time.Second, time.Minute, func() (bool, error) {
+		// Non-expiring defaultMeter should be available
+		return len(allMeters.meters) == 2, nil
```

Comment:
well that's just one — what's the second one?
Also I'd suggest 100ms/5s params for poll

Reply:
Second one?

Reply:
There is the default one and one that comes from the test.